C/ds/minHeap.c

```diff
@@ -0,0 +1,192 @@
+/*
+A min heap has the property that the parent is less than or equal to the child nodes. 
+When using a 0-indexed array,
+Left child of a node at i is (2 * i + 1)
+Right child of a node at i is (2 * i + 2)
+Parent is (i - 1) / 2
+*/
```

Comment:
Use multiline comments `//` instead of single line comments when you're writing n line comments.

Reply:
Okay. But I have used multi-line comments only right? 😓